tests/test_enterprise/test_utils.py

```diff
@@ -33,15 +33,15 @@ class StubModel(Exception):
     pass
 
 
-@mark.django_db
-@ddt.ddt
+@ mark.django_db
+@ ddt.ddt
```

Comment:
Are these supposed to have spaces after the `@`?

Reply:
oh wow I think my auto linter did that 😮

Reply:
sure sure, "my auto linter ate my decorators"